dbt/include/teradata/macros/materializations/incremental/strategies.sql

```diff
@@ -134,4 +134,191 @@
             {%- endfor %}
         )
 
+{% endmacro %}
+
+{% macro drop_staging_tables_for_valid_history(table_names) %}
+    {% for table_name in table_names %}
+        {% call statement('dropping existing staging table ' ~ table_name) %}
+            DROP table /*+ IF EXISTS */ {{ table_name }};
+        {% endcall %}
+    {% endfor %}
+{% endmacro %}
+
+{% macro create_staging_tables_for_valid_history(table_names, target) %}
+    {% for table_name in table_names %}
+        {% call statement('creating staging table ' ~ table_name) %}
+            create set table {{ table_name }} as {{ target }} with no data;
+        {% endcall %}
+    {% endfor %}
+{% endmacro %}
+
+{% macro teradata__get_incremental_valid_history_sql(target, source, unique_key, valid_period, valid_from, valid_to, use_valid_to_time, history_column_in_target, resolve_conflicts) -%}
+--    {{ log("**************** in teradata__get_incremental_valid_history_sql macro")  }}

```

Comment:
@tallamohan 
Are all of these log statements required?

Reply:
These logs are commented